tests/api/api.spec.ts

```diff
@@ -1116,16 +1116,15 @@ select \${foo}
       expect(dataform.RunResult.create(cleanTiming(await runner.execute().result()))).to.deep.equal(
         EXPECTED_RUN_RESULT
       );
-      verify(mockedDbAdapter.createSchema("database", "schema1")).once();
-      verify(mockedDbAdapter.createSchema("database2", "schema2")).once();
+      verify(mockedDbAdapter.createSchema(anything())).twice();
```

Comment:
Is there a way to catch match protos here so don't need to make it more ambiguous?

Reply:
have you tried both `{database: ..., schema: ...}` and `dataform.Schema.create({ database: ..., schema: ...})` forms?